### PR TITLE
Feature/unnullify values

### PR DESCRIPTION
**Pull request by jacob-andersson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,12 +12,13 @@
 ```objc
 // Converts all the keys in the JSON to camelCase
 - (id)minced_JSONKeysToCamelCase;
-
-// Converts all the keys in the NSArray to camelCase
 - (NSArray *)minced_JSONObjectsKeysToCamelCase;
-
-// Converts all the keys in the NSDictionary to camelCase
```
Comment (3lvis) [inline README.md]: Any reason to remove this?

File: README.md [hunk 2]
```diff
@@ -12,12 +12,13 @@
 ```objc
 // Converts all the keys in the JSON to camelCase
 - (id)minced_JSONKeysToCamelCase;
-
-// Converts all the keys in the NSArray to camelCase
```
Comment (3lvis) [inline README.md]: Any reason to remove this?

File: README.md [hunk 3]
```diff
@@ -12,12 +12,13 @@
...
 - (NSArray *)minced_JSONObjectsKeysToCamelCase;
-
-// Converts all the keys in the NSDictionary to camelCase
 - (NSDictionary *)minced_JSONObjectKeysToCamelCase;
+
+// Converts all the keys in the JSON to camelCase and replaces null values with ""
```
Comment (3lvis) [inline README.md]: Might be better if we say

// Converts all the keys in the JSON to camelCase and replaces null values with an empty string

File: Source/NSObject+HYPMinced.m [hunk 4]
```diff
@@ -4,44 +4,80 @@
...
+{
     if ([self isKindOfClass:NSArray.class]) {
-        return [self minced_JSONObjectsKeysToCamelCase];
+        return [self minced_JSONObjectsKeysToCamelCaseWithValuesUnnullified:unnullify];
     } else if ([self isKindOfClass:NSDictionary.class]) {
-        return [self minced_JSONObjectKeysToCamelCase];
+        return [self minced_JSONObjectKeysToCamelCaseWithValuesUnnullified:unnullify];
     }
-
+
```
Comment (3lvis) [inline Source/NSObject+HYPMinced.m]: Please trim this whitespaces

**Comment by 3lvis** [comment]
Looks good, can you also ignore the Pods folder and remove it from git?

**Comment by 3lvis** [comment]
Sweet work @jacob-andersson!

**Comment by 3lvis** [comment]
Is this the Realm thing? Maybe we can add some nice section to README about it and share it on our Twitter account.

**Comment by jacob-andersson** [comment]
@3lvis: Yes, it is THE REALM THING...

**Pull request merged**